tests/data/export_coco_as_yolov5/data.yml

```diff
@@ -0,0 +1,7 @@
+names:
```

Comment:
there are lots of unnecessary files :(

Reply:
Sure, I'll fix unnecessary file conflicts and open a new one. After it's added someone can figure how to implement NMS/NMM/GREEDYNMM for polygons efficiently.

Reply:
You can ignore the polygon postprocess in your pr, I will try to handle it in a separate pr 👌🏻